applications/asset_tracker_v2/src/led/led.c

```diff
@@ -102,8 +102,8 @@ int led_init(void)
 		return err;
 	}
 
-	k_delayed_work_init(&leds_update_work, leds_update);
-	k_delayed_work_submit(&leds_update_work, K_NO_WAIT);
+	k_work_init_delayable(&leds_update_work, leds_update);
+	k_work_reschedule(&leds_update_work, K_NO_WAIT);
```

Comment:
Can be `schedule()`, same goes for calls in the handler

Reply:
done